src/object/cli_obj.c

```diff
@@ -1726,19 +1726,15 @@ dc_obj_retry_delay(tse_task_t *task, int err, uint16_t *retry_cnt, uint16_t *inp
 	 * -DER_INPROGRESS || -DER_UPDATE_AGAIN cases.
 	 */
 	++(*retry_cnt);
-	if (err == -DER_INPROGRESS || err == -DER_UPDATE_AGAIN) {
-		if (++(*inprogress_cnt) > 1) {
-			limit += *inprogress_cnt;
-			if (limit > 10)
-				limit = 10;
+	if (++(*inprogress_cnt) > 1) {
```

Comment:
Please increase `inprogress_cnt` only for "if (err == -DER_INPROGRESS || err == -DER_UPDATE_AGAIN)" to avoid confusing in the log message.

Reply:
Do you actually mean the comment? I tend to make this generic. I can fix the comment though

Reply:
No comment, I mean that:
`
if (err == -DER_INPROGRESS || err == -DER_UPDATE_AGAIN)
++(*inprogress_cnt);
if (++(*retry_cnt) > 1) {
...
}`

Reply:
I see. In that case, we should take the number of retry_cnt into consideration.